common/lanelet2_extension/lib/autoware_osm_parser.cpp

```diff
@@ -137,7 +122,13 @@ void AutowareOsmParser::parseMapParams (const std::string& filename, int* projec
     }
   }
   else
-    throw lanelet::ParseError(std::string("While parsing .osm file, target_frame could not be found in geoReference tag."));
+  {
+    throw lanelet::ParseError(std::string("While parsing .osm file, geoReference was not found!"));
+  }
+
+  // Default values
+  *projector_type = 1; // default value for autoware.ai projector type for CARMA purposes
+  *base_frame = std::string("+proj=geocent +ellps=WGS84 +datum=WGS84 +units=m +no_defs"); //default value for ECEF proj string
```

Comment:
I was originally hoping that these could be ROS parameters with these values set as defaults. Having a constant in the function makes the parameters to the function largely unnecessary. But I can also see that the usage of projector type 0 would be incompatible with carma anyway since there is currently no way for us to publish the ecef transform. 

I guess I am fine with this then. Though it seems like the role of this function is somewhat mute now.